### DISCUSSION TITLE
PpocrLabel和vi_layoutxlm 是否有长度限制

### DISCUSSION THREAD

**Discussion by yuhakusho** [PaddlePaddle/PaddleOCR#13825] [Q&A]
1、PpocrLabel自动标注是否有长度限制，再对一个图片上的表格进行自动标注时候只标注了上半页（表格数字比较多）。
哪里可以修改。

2、vi_layoutxlm 训练的ser模型有两个关键词检测不到，当我对图片其它区域涂抹以后可以检测到。是否对文本有长度限制？如何修改？ 通过改config目录下配置会报错

**Comment by UserWangZz** [comment]
首先，表格识别目前仍然是hard任务，PPOCRLabel中使用SLANet模型来进行表格识别，但是对于复杂表格而言，识别效果可能不及预期，因此需要人工手动调整。
ser和re模型已经是过时的算法了，如果对KIE任务有需求，我们更推荐使用PPChatOCR，它将PaddleOCR与百度文心大模型结合，利用大模型的优秀语言能力进行KIE提取。
